apps/microservices-orders/orders-service/src/main/java/io/confluent/kafkadevops/microservicesorders/ordersservice/SchemaRegistryConfig.java

```diff
@@ -0,0 +1,37 @@
+package io.confluent.kafkadevops.microservicesorders.ordersservice;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+@Component
+@Scope("singleton")
+public class SchemaRegistryConfig {
```

Comment:
why we need this? spring-kafka support configuring  SR in property files

Reply:
> why we need this? spring-kafka support configuring SR in property files

There was a DI dependency issue i'm struggling to remember the need for this.   It may have been related to 'mock://' SR and the timing of creation of Kafka Streams and serializer creations, but I cannot recall the foundations of my class design at this time.

Reply:
@gAmUssA, I'm fairly certain the reason I did this was that I needed to configure the Kafka Streams Serde (for example: https://github.com/confluentinc/kafka-devops/pull/32/files#diff-271f51bab3aee5d6b0c6a707ee998ae409c754e9b752773e323b0abae7e25c57R33).   IIUC Kafka Streams is picking up kafka configurations from the prop files, but this Serde will not, requiring "manual" configuration.  Please let me know if I'm misunderstanding, the framework is still very new to me.

So, if I need to manually configure, I wanted to create a singleton that could receive the configurations, and that could be injected into classes that need it.  This could reduce boilerplate (only one place for `@Value` attributes for SR), centralize the SR configs, etc...   Please set me straight if I'm missing another way.  Thanks for your review.